launcher/launcher_test.go

```diff
@@ -292,7 +292,7 @@ var _ = Describe("Launcher", func() {
 
 					It("returns an error", func() {
 						Eventually(session).Should(gexec.Exit(4))
-						Eventually(session.Err).Should(gbytes.Say("Unable to verify CredHub server"))
+						Eventually(session.Err).Should(gbytes.Say("Unable to interpolate credhub references"))
```

Comment:
Leaving this as a note for posterity: it looks this test has been broken for years, but this packages is being skipped in CI 😱 . I submitted a PR to fix that: https://github.com/cloudfoundry/diego-release/pull/728